app/models/protocol.rb

```diff
@@ -164,6 +163,16 @@ class Protocol < ActiveRecord::Base
       where.not(project_roles: { project_rights: 'none' })
   }
 
+  scope :admin_filter, -> (params) {
```

Comment:
This filter breaks apart the input string to determine which filter to use and the ID to pass